apiserver/worker/compiler.py

```diff
@@ -412,7 +416,16 @@ def compile(self, bot_dir, globs, errors, timelimit):
     ),
     Language("Dart", BOT +".dart", "MyBot.dart",
         "dart MyBot.dart",
-        [], [(["*.dart"], ChmodCompiler("Dart"))]),
+        [], [(["*.dart"], ChmodCompiler("Dart"))]
+    ),
+    Language("Elixir", BOT, "mix.exs",
+        "./MyBot",
+        [],
+        [
+            ([], ErrorFilterCompiler(com_args["Elixir"][0])),
```

Comment:
Small typo (my bad) - should be `comp_args` not `com_args` here and below

Reply:
Cool, I'll fix these and squash the commits so this doesn't look so noisy when someone looks thru history.